nectar/downloaders/threaded.py

```diff
@@ -182,16 +187,14 @@ def _download_one(self, request):
         :return:    download report
         :rtype:     nectar.report.DownloadReport
         """
-        session = build_session(self.config)
-        return self._fetch(request, session)
+        build_session(self.config, self.session)
```

Comment:
I realize that this does the work in place, but since the return value is used elsewhere, do you think `self.session = build_session(whatever)` is more clear?